Remove customized handling of ${CMAKE_BUILD_TYPE} which conflicts with
how Gentoo handles ${CMAKE_BUILD_TYPE}:
* CMake Error at cmake/modules/CGAL_SetupFlags.cmake:65 (message):
*   Gentoo is not a valid build type: only Release or Debug is allowed
* Call Stack (most recent call first):
*   CMakeLists.txt:519 (include)

--- CGAL-4.7/cmake/modules/CGAL_SetupFlags.cmake
+++ CGAL-4.7/cmake/modules/CGAL_SetupFlags.cmake
@@ -61,9 +61,6 @@
   add_definitions(-DCGAL_TEST_SUITE)
 endif()
 
-if ( NOT "${CMAKE_BUILD_TYPE}" STREQUAL "Release" AND NOT "${CMAKE_BUILD_TYPE}" STREQUAL "Debug" )
-  message( FATAL_ERROR "${CMAKE_BUILD_TYPE} is not a valid build type: only Release or Debug is allowed" )
-endif()
 
 message( STATUS "Build type: ${CMAKE_BUILD_TYPE}" )
 
